src/linker/Linker.Dataflow/ArrayBuilder.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
```

Comment:
License header

Reply:
This should be in Mono.Linker there is nothing .Dataflow specific and can be used elsewhere